project_setup/Skeletonize-existing-repository.md

```diff
@@ -0,0 +1,84 @@
+# Skeletonize an existing repository #
+
+Skeletonizing a repository standardizes our development setup and environment
+as well as enables our [Lineage](https://github.com/cisagov/action-lineage/)
+Github Action to keep it updated and standardized.
+
+## About skeletons ##
+
+Skeleton projects contain [licensing information](LICENSE), as
+well as [pre-commit hooks](https://pre-commit.com) and
+[GitHub Actions](https://github.com/features/actions) configurations
+appropriate for the major languages that we use. This lets us standardize
+[cisagov](https://github.com/cisagov) GitHub projects to a
+[list of cisagov skeleton projects](https://github.com/search?q=org%3Acisagov+skeleton&type=Repositories).
+
+## General overview ##
+
+The general outline of how to add a skeleton to a repository is:
+
+1. Add the skeleton as a remote to the non-skeletonized repository
+2. Pull with `--allow-unrelated-histories`
+3. Fix all the inevitable conflicts (this takes the longest)
+4. Run `pre-commit install`
+5. Fix problems that may arise from `pre-commit` being added
+6. Commit to a branch and submit a PR
+
+## Adding a skeleton as remote ##
+
+### Choose a skeleton ###
+
+First, decide which of the available skeletons fits your existing repository.
```

Comment:
Do we want to mention the `skeleton list` command, which outputs all of our existing skeletons?

Reply:
Added in https://github.com/cisagov/development-guide/pull/32/commits/6234992a97b35c2ab7b7286149b01402ec07ed9e